src/components/Inputs/CustomDatePicker.js

```diff
@@ -22,6 +22,8 @@ const CustomDatePicker = ({
   error,
   helperText,
   disabledRangeDate = {},
+  max = null,
+  min = null,
```

Comment:
why do we need to change in shared component?? can't you apply directly min and max?? using disabledRangeDate??